build/edge/run_daemon.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+if [ ! -d /etc/kubeedge/edge/certs ] || [ ! -e /etc/kubeedge/edge/certs/edge.crt ] || [ ! -e /etc/kubeedge/edge/certs/edge.key ]; then
+    mkdir -p /etc/kubeedge/edge/certs
+    mkdir -p /etc/kubeedge/ca
+    docker run --rm \
+    -v /etc/kubeedge/ca:/etc/kubeedge/ca \
+    -v /etc/kubeedge/edge/certs:/etc/kubeedge/certs \
+    kubeedge/certgen:v0.2 genCertAndKey edge
+fi
+
+if [ ! -d /var/lib/kubeedge ]; then
+    mkdir -p /var/lib/kubeedge
+fi
+
+if [ ! -d /var/lib/edged ]; then
+    mkdir -p /var/lib/edged
+fi
+
+if [ ! -S /var/run/docker.sock ]; then
+    echo "docker engine not found"
+    exit -1
+fi
+
+start() {
+    local mqttBroker=$1
+    local cloudHub=$2
+    docker run -d --name edgecore --restart always \
+    --cpu-period=50000 --cpu-quota=100000 --memory=1g --privileged \
+    -e mqtt.server=${mqttBroker} \
+    -e edgehub.websocket.url=${cloudHub} \
+    -v /etc/kubeedge/edge/certs:/etc/kubeedge/edge/certs:ro \
+    -v /var/lib/edged:/var/lib/edged \
+    -v /var/lib/kubeedge:/var/lib/kubeedge \
+    -v /var/run/docker.sock:/var/run/docker.sock \
+    kubeedge/edgecore:v0.2
```

Comment:
the Makefile uses the ${IMAGE_TAG}  to tag the edgecore docker image.
```sh
docker build -t kubeedge/edgecore:${IMAGE_TAG} -f build/edge/Dockerfile .
```
This script uses a hard-coded version v0.2 ? can we pass the image name as a parameter ?

Reply:
OK